Modules/_tracemalloc.c

```diff
@@ -1259,6 +1269,13 @@ trace_to_pyobject(unsigned int domain, trace_t *trace,
     }
     PyTuple_SET_ITEM(trace_obj, 2, obj);
 
+    obj = PyLong_FromSize_t(trace->traceback->length);
```

Comment:
nitpick: I would prefer to use PyLong_FromLong() which is more common and closer to int type.

Reply:
ok!